docs/targets/zendesk.md

```diff
@@ -0,0 +1,68 @@
+# Event target for Zendesk
+
+This event target recieves [CloudEvents][ce] and interfaces with Zendesk by either creating a new Ticket or placing a new Tag on a Ticket depending on the CloudEvent type.
+
+## Prerequisite: API token
+
+A Zendesk API key is required to utilize this target. The steps to obtain a key are outlined in the [Zendesk API Docs](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-)
+
+## Deploying an instance of the Target
+
+Open the Bridge creation screen and add a target of type `Zendesk`.
+
+![Adding a Zendesk target](../images/zendesk-target/create-bridge-1.png)
+
+In the Target creation form, give a name to the event Target and add the following information:
+
+* **Email**: Email address associated with the Zendesk account.
+* [**Subdomain**][zd-subdom]: Name of the Zendesk subdomain, without the `zendesk.com` domain or `https://` scheme.
+* [**Token**][zd-token]: Reference to a [TriggerMesh secret][tm-secret] containing an API token to communicate with the Zendesk API, as described in the previous section.
+* **Subject**: An optional default subject assignment for new tickets
```

Comment:
You may want to highlight what the default subject would be if one is not provided.